src/Nest/Indices/MappingManagement/PutMapping/ElasticClient-Map.cs

```diff
@@ -2,7 +2,7 @@
 // Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information
 
-﻿using System;
+ using System;
```

Comment:
This space issue is quite annoying, we might need to remove the insert license from our formatting profile.